dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/TaskDefinitionMapper.xml

```diff
@@ -86,6 +86,14 @@
         delete from t_ds_task_definition
         where code = #{code}
     </delete>
+
+    <delete id="deleteByCodeList">
+        delete from t_ds_task_definition
+        where code in
+        <foreach collection="codeList" item="code" separator="," open="(" close=")">
+            #{code}
+        </foreach>
+    </delete>
```

Comment:
We should also delete the `t_ds_task_definition_log`.

Reply:
It looks like process only delete the definition, do we need to unified?